### PR TITLE
netgauge: Fix for aarch64

### PR DESCRIPTION
**Pull request by t-nojiri**
I made a fix to be able to build on aarch64.

### CODE REVIEW

File: var/spack/repos/builtin/packages/netgauge/package.py [hunk 1]
```diff
@@ -18,3 +18,10 @@ class Netgauge(AutotoolsPackage):
     version('2.4.6', sha256='dc9398e4e042efec70881f2c7074ff18cc5b74bc5ffc4b8a4aaf813b39f83444')
 
     depends_on("mpi")
+
+    def configure_args(self):
+        args = []
+        if self.spec.target.family == 'aarch64':
+            args.append('MPICC=mpicc')
+            args.append('MPICXX=mpicxx')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/netgauge/package.py]: I wonder if we shouldn't use `self.spec['mpi'].mpicc` (the absolute path to the mpi wrapper, independent of mpi library) and do this for all platforms...

**Comment by t-nojiri** [comment]
The homepage (http://unixer.de/research/netgauge/) states that MPICC = mpicc MPICXX = mpicxx is specified in configure.

I fixed it as below, but the compiler was not recognized.
    args.append('MPICC=%s' % self.spec['mpi'].mpicc)
    args.append('MPICXX=%s' % self.spec['mpi'].mpicxx)

**Comment by alalazo** [comment]
@t-nojiri Did you try also:
```console
args.append('MPICC=%s' % os.path.basename(self.spec['mpi'].mpicc))
args.append('MPICXX=%s' % os.path.basename(self.spec['mpi'].mpicxx))
```
?

**Comment by t-nojiri** [comment]
I was able to build it.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**